testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/UpdateCustomConditionsTest.java

```diff
@@ -0,0 +1,192 @@
+package io.stargate.it.http.graphql.graphqlfirst;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class UpdateCustomConditionsTest extends GraphqlFirstTestBase {
+
+  private static CqlSession SESSION;
+  private static GraphqlFirstClient CLIENT;
+  private static String KEYSPACE;
+
+  private static String getUserName(int pk) {
+    ResultSet resultSet = SESSION.execute("SELECT * FROM \"User\" WHERE pk = ? ", pk);
+    return Objects.requireNonNull(resultSet.one()).getString("username");
+  }
+
+  private static Row getUserRow(int pk) {
+    ResultSet resultSet = SESSION.execute("SELECT * FROM \"User\" WHERE pk = ? ", pk);
+    return resultSet.one();
+  }
+
+  @BeforeAll
+  public static void setup(
+      StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId, CqlSession session) {
+    SESSION = session;
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    KEYSPACE = keyspaceId.asInternal();
+    CLIENT.deploySchema(
+        KEYSPACE,
+        "type User @cql_input {\n"
+            + "  pk: Int! @cql_column(partitionKey: true)\n"
+            + "  username: String"
+            + "  age: Int\n"
+            + "}\n"
+            + "type Query { user(pk: Int!): User }\n"
+            + "type UpdateUserResponse @cql_payload {\n"
+            + "  applied: Boolean\n"
+            + "  user: User!\n"
+            + "}\n"
+            + "type Mutation {\n"
+            + "  updateUser(user: UserInput!): UpdateUserResponse @cql_update(targetEntity: \"User\")\n"
+            + "  updateUserEQCustomPayload(\n"
+            + "    pk: Int\n"
+            + "    username: String\n"
+            + "    age: Int @cql_if(field: \"age\", predicate: EQ)\n"
+            + "  ): UpdateUserResponse @cql_update(targetEntity: \"User\")\n"
+            + "  updateUserIfExists(user: UserInput!): UpdateUserResponse @cql_update\n"
+            + "  updateUserIfExistsCustomPayload(user: UserInput!): UpdateUserResponse @cql_update(ifExists: true)\n"
```

Comment:
Could you add a test with WHERE...IN conditions?
(I assume your comment below refers to IF conditions)

Reply:
added test for IN